open_spiel/games/quoridor_test.cc

```diff
@@ -34,15 +34,28 @@ void BasicQuoridorTests() {
         *LoadGame(absl::StrCat("quoridor(board_size=", i, ")")), 5);
   }
 
+  for (int i = 2; i <= 2; i++) {
```

Comment:
A loop from i=2 to i<=2? Did you mean i <= 4 here?

Reply:
I've imported this and fixed this to <= 4. No need to update. It's undergoing internal review now, I'll add the "imported" label when it's done.

Reply:
Ah yes, it was meant to test all possible numbers of players. Thank you! ;)